modules/apim_common/manifests/params.pp

```diff
@@ -19,6 +19,10 @@
   $packages = ["unzip"]
   $version = "2.6.0"
 
+  $pack_location = "local"
+  # $pack_location = "remote"
```

Comment:
Shall we add a comment explaining what `pack_location` is about?

Reply:
Fixed https://github.com/wso2/puppet-apim/pull/128/commits/775a688e3eae96b3f6c013f85e1fed7bd19ae827